frontend/Utils/geolocation.js

```diff
@@ -0,0 +1,77 @@
+import Geocode from 'react-geocode';
+
+import {
+  MAP_KEY,
+  MAP_DEFAULT_CENTER
+} from '../../config/credentials';
+
+import _ from 'lodash';
+
+// geocode configs
+Geocode.setApiKey(MAP_KEY);
+Geocode.setLanguage('en');
+
+// Credit to https://www.w3.org/2003/01/geo/test/ustowns/latlong.htm for this data
+const defaultCentersMap = {
```

Comment:
probably want to move this out into some sort of .json (ideally so the board admins can just edit a json instead of editing the JS code)

Reply:
yeah great idea! I just hacked this object up as a POC, this looks like a great substitute!